content/get-started/getting-started-with-git/configuring-git-to-handle-line-endings.md

```diff
@@ -107,29 +107,40 @@ You'll notice that files are matched—`*.c`, `*.sln`, `*.png`—, separated by
 
 When you set the `core.autocrlf` option or commit a *.gitattributes* file, you may find that Git reports changes to files that you have not modified. Git has changed line endings to match your new configuration.
 
-To ensure that all the line endings in your repository match your new configuration, backup your files with Git, delete all files in your repository (except the `.git` directory), then restore the files all at once.
-
-1. Save your current files in Git, so that none of your work is lost.
-  ```shell
-  $ git add . -u
-  $ git commit -m "Saving files before refreshing line endings"
-  ```
-2. Add all your changed files back and normalize the line endings.
-  ```shell
-  $ git add --renormalize .
-  ```
-3. Show the rewritten, normalized files.
-  ```shell
-  $ git status
-  ```
-4. Commit the changes to your repository.
-  ```shell
-  $ git commit -m "Normalize all the line endings"
-  ```
+To ensure that all the line endings in your repository match your new configuration, backup your files with Git, then remove/restore the files all at once with renormalized line endings.
+
+1. Save your current files in Git, so that none of your work is lost.  
+   ```shell
+   $ git add . -u
+   $ git commit -m "Saving files before refreshing line endings"
+   ```
+2. Add/re-add any new/changed files and normalize the line endings.  
+   ```shell
+   $ git add --renormalize .
+   ```  
+   Or, alternatively, if you just need to convert all files in the HEAD of an existing checked-out repository to the new git line-ending configuration:  
+   a) Remove existing cached file-entries for current checked out branch.  
+   ```shell
+   $ git rm -rf --cached .
+   ```  
+   b) Reset the repository HEAD to the last commit, overwriting the existing files using the git repository's new line-endings.  
+   ```shell
+   $ git reset --hard HEAD
+   ```  
+   (In that case, if no further changes to the repo are required, steps 3 & 4 can be skipped.)
+3. Show the rewritten, normalized files.  
+   ```shell
+   $ git status
+   ```
+4. Commit the changes to your repository.  
+   ```shell
+   $ git commit -m "Normalize all the line endings"
+   ```
 
 ## Further reading
 
 - [Customizing Git - Git Attributes](https://git-scm.com/book/en/Customizing-Git-Git-Attributes) in the Pro Git book
 - [git-config](https://git-scm.com/docs/git-config) in the man pages for Git
 - [Getting Started - First-Time Git Setup](https://git-scm.com/book/en/Getting-Started-First-Time-Git-Setup) in the Pro Git book
 - [Mind the End of Your Line](http://adaptivepatchwork.com/2012/03/01/mind-the-end-of-your-line/) by [Tim Clem](https://github.com/tclem)
+- [Stackoverflow: "How do I force git to use LF instead of CR+LF under windows?"](https://stackoverflow.com/questions/2517190/how-do-i-force-git-to-use-lf-instead-of-crlf-under-windows)
```

Comment:
I appreciate that this addition functions as a kind of attribution, but we don't typically link to discussions on Stack Overflow from our product documentation.

```suggestion
```